Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackForm.swift

```diff
@@ -432,32 +510,51 @@ class SentryUserFeedbackForm: UIViewController {
 // MARK: UITextFieldDelegate
 @available(iOS 13.0, *)
 extension SentryUserFeedbackForm: UITextFieldDelegate {
-    func textFieldDidBeginEditing(_ textField: UITextField) {
-        editingTextField = textField
-        editingTextView = nil
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+        textField.resignFirstResponder()
+        return true
     }
 }
 
 // MARK: UITextViewDelegate
 @available(iOS 13.0, *)
 extension SentryUserFeedbackForm: UITextViewDelegate {
     func textViewDidChange(_ textView: UITextView) {
-        editingTextField = nil
-        editingTextView = textView
         messageTextViewPlaceholder.isHidden = textView.text != ""
     }
 }
 
 // MARK: UIImagePickerControllerDelegate & UINavigationControllerDelegate
 @available(iOS 13.0, *)
 extension SentryUserFeedbackForm: UIImagePickerControllerDelegate & UINavigationControllerDelegate {
+    static let formatter = {
+        let formatter = DateFormatter()
+        formatter.dateStyle = .medium
+        formatter.timeStyle = .medium
+        return formatter
+    }()
+    
     func imagePickerController(_ picker: UIImagePickerController, didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey: Any]) {
+        defer {
+            dismiss(animated: config.animations)
+        }
+        
         guard let photo = info[.editedImage] as? UIImage else {
             // TODO: handle error
             return
         }
         addedScreenshot(image: photo)
-        dismiss(animated: config.animations)
+         
+        guard let asset = info[.phAsset] as? PHAsset else {
+            // TODO: handle error
+            return
+        }
+        
+        guard let date = asset.creationDate else {
+            // TODO: handle error
```

Comment:
`l`: How did this `TODO:` pass the pre-commit hooks? Shouldn't it be detected by SwiftLint?

Reply:
I have that swiftlint rule disabled for some of these implementations

Reply:
i'll take care of this one as it should probably just be another log, but I also added a checklist item to remove the overrides before shipping: #4709